src/pages/california-privacy-notice.md

```diff
@@ -0,0 +1,286 @@
+---
+template: legal-page
+path: /california-privacy-notice
+title: KEEP SECZ Privacy Notice for California Residents
+date: "Effective Date: April 16, 2020"
+intro: >-
+  <p>
+    This <strong>Privacy Notice for California Residents</strong>&nbsp;
+    supplements the information contained in KEEP SECZ&apos;s&nbsp;
+    <a href="/privacy-policy">Privacy Policy</a> and applies solely to all
+    visitors, users, and others who reside in the State of California
+    (&ldquo;consumers&rdquo; or &ldquo;you&rdquo;). We adopt this notice
+    to comply with the California Consumer Privacy Act of 2018 (CCPA) and
+    any terms defined in the CCPA have the same meaning when used in this
+    notice.
+  </p>
+---
+## Information We Collect
+
+Our Website may collect information that identifies, relates to, describes, references, is capable of being associated with, or could reasonably be linked, directly or indirectly, with a particular consumer or device (&ldquo;**personal information**&rdquo;). In particular, our Website collects the following categories of personal information from its consumers within the last twelve (12) months:
+
+<table>
```

Comment:
One useful “later” thing given the context in which these are edited in the CMS would be to turn these into [Markdown tables](https://help.github.com/en/github/writing-on-github/organizing-information-with-tables).